client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -198,6 +205,8 @@ const RecurringPaymentsPlanAddEditModal = ( {
 					multiple_per_user: editedMultiplePerUser,
 					welcome_email_content: editedCustomConfirmationMessage,
 					subscribe_as_site_subscriber: editedPostsEmail,
+					type: editedMarkAsDonation,
+					public: true,
```

Comment:
Update the public field name as spoken here: https://github.com/Automattic/jetpack/pull/23590